locations/spiders/arbys.py

```diff
@@ -0,0 +1,129 @@
+# -*- coding: utf-8 -*-
+import json
+import scrapy
+from urllib import parse
+from locations.items import GeojsonPointItem
+
+STATES = ["AL", "AK", "AZ", "AR", "CA", "CO", "CT", "DC", "DE", "FL", "GA",
```

Comment:
Instead of using their search page, it'll probably be more efficient to spider through the directory of all stores present at https://locations.arbys.com/.

Reply:
@iandees  Ok so instead of searching go to the page that lists all of them?

Reply:
Yep, that should make it a little faster. Let me know if you have any questions.

Reply:
@iandees sure! Not a problem, I will handle it.